.nvmrc

```diff
@@ -1 +1 @@
-8.2.1
+8
```

Comment:
How come?

Reply:
To pick up compatible changes w/o needing to track Node releases closely and changing this when they occur. Pin it to `8.2` if you want, but locking to a patch release prevents bug / security fixes from being picked up.

Reply:
Alternately, is there a reason that specifically 8.2.1 needs to be used?

Reply:
Isn't this the same conversation that led to `npm` finally adding `package-lock.json`? Sure the flexibility of `^1`-style versioning makes security patching "easier" (you still have to rebuild the project, it's not total magic), but you lose out on the main benefit of version pinning: behaviour guarantees. Which is why `npm` relented and made `package-lock.json` default in v5.

So `8.2.1` is a guarantee of behaviour and passing tests.

Reply:
Feel free to revert this.